multiqc/modules/fastqc/fastqc.py

```diff
@@ -312,11 +312,12 @@ def read_count_plot(self):
             pd = self.fastqc_data[s_name]["basic_statistics"]
             pdata[s_name] = dict()
             try:
-                pdata[s_name]["Duplicate Reads"] = int(
-                    ((100.0 - float(pd["total_deduplicated_percentage"])) / 100.0) * pd["Total Sequences"]
-                )
-                pdata[s_name]["Unique Reads"] = pd["Total Sequences"] - pdata[s_name]["Duplicate Reads"]
-                has_dups = True
+                if not math.isnan(float(pd["total_deduplicated_percentage"])):
+                    pdata[s_name]["Duplicate Reads"] = int(
+                        ((100.0 - float(pd["total_deduplicated_percentage"])) / 100.0) * pd["Total Sequences"]
+                    )
+                    pdata[s_name]["Unique Reads"] = pd["Total Sequences"] - pdata[s_name]["Duplicate Reads"]
+                    has_dups = True
             except KeyError:
```

Comment:
Bit late now sorry, but this would have been a simpler fix 😉 

```suggestion
            except (KeyError, ValueError):
```